### PR TITLE
Fixed Loan Action Button UI

### LINKED ISSUE #3186: Loan Actions Button UI can be improved
Loan actions buttons are not aligned properly and can be styled to enhance UI
### Evidence / Screenshot (if possible)
Screenshot_20200315_011353
### Relevant url?

https://openlibrary.org/account/loans
### Steps to Reproduce

1. Go to https://openlibrary.org/account/loans
2. You should have borrowed at least 2 books in order to see the loan action buttons

* Actual: Poorly aligned buttons 
* Expected: Proper styled buttons that are easy to understand.

### Details

- **Logged in (Y/N)?** Y
- **Browser type/version?**  Chrome

- **Operating system?** Ubuntu 19.10
- **Environment (prod/dev/local)?** prod

### Proposal & Constraints

Just have to style the buttons by putting the appropriate class name if it already exists.
### Related files

Not sure about the right CSS file please add it here if possible.

### PR DESCRIPTION
**Pull request by the-nine-tail**
Second Commit In OpenLibrary - 17032020
This commit fix the loan button ui, before the read online and return button where overlapping and also where not responsive

The file LoadReadForm and ReturnForm has been modified to make the button responsive for all devices

Closes #3186

### CODE REVIEW

File: ReturnForm.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+$def with (loan)
+
+$# "do_return" is a fake "title" for the book that can be used for stats
+$ return_url = loan['book'] + '/do_return/borrow'
+
+<style type="text/css">
+form.returnbook {
```
Comment (jdlrobson) [inline ReturnForm.html]: Inline styles cause maintenance problems so we prefer to keep all CSS inside static/css

Please move these files to a file that is referenced in the entry point.
The entry point file you'll want to add these rules to is static/css/page-user.less

File: LoanReadForm.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+$def with(edition_key)
+
+<style type="text/css">
+.darkgreen.local-date-time {
```
Comment (jdlrobson) [inline LoanReadForm.html]: Inline styles cause maintenance problems so we prefer to keep all CSS inside static/css

Please move these files to a file that is referenced in the entry point.
The entry point file you'll want to add these rules to is static/css/page-user.less

File: LoanReadForm.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+$def with(edition_key)
```
Comment (jdlrobson) [inline LoanReadForm.html]: Templates live inside the openlibrary/templates folder.

File: LoanReadForm.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    position: relative;
+    top: 0.6em;
+    left: 3em;
+}
+</style>
+
+<form class="LoanReadForm" method="POST" action="$edition_key/_doread/borrow"><input type="hidden" name="action" value="read" /><input type="text" name="ol_host" value="" class="hidden ol_host" /><input type="submit" value="Read Online" /></form><br />
+
+<script type="text/javascript">
+    jQuery('.ol_host').attr('value', location.host);
```
Comment (jdlrobson) [inline LoanReadForm.html]: Javascript should live inside openlibrary/plugins/openlibrary/js/

File: LoanReadForm.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    left: 2.8em;
+  }
+form.LoanReadForm {
+    position: relative;
+    top: 0.6em;
+    left: 3em;
+}
+</style>
+
+<form class="LoanReadForm" method="POST" action="$edition_key/_doread/borrow"><input type="hidden" name="action" value="read" /><input type="text" name="ol_host" value="" class="hidden ol_host" /><input type="submit" value="Read Online" /></form><br />
```
Comment (tfmorris) [inline LoanReadForm.html]: The label "Read Online" needs to be internationalized.

File: ReturnForm.html [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    position: relative;
+    left: 3em;
+}
+</style>
+<form class="returnbook" action="$return_url" method="post">
+<input type="submit" value="Return book" />
```
Comment (tfmorris) [inline ReturnForm.html]: Another missing i18n - "Return book"

File: ReturnForm.html (hunk 1 above)
Comment (the-nine-tail) [reply ReturnForm.html]: Done

File: LoanReadForm.html (hunk 2 above)
Comment (the-nine-tail) [reply LoanReadForm.html]: Done

File: LoanReadForm.html (hunk 5 above)
Comment (mekarpeles) [inline LoanReadForm.html]:
```suggestion
<form class="LoanReadForm" method="POST" action="$edition_key/_doread/borrow"><input type="hidden" name="action" value="read" /><input type="text" name="ol_host" value="" class="hidden ol_host" /><input type="submit" value="$_('Read Online')" /></form><br />
```

File: ReturnForm.html (hunk 6 above)
Comment (mekarpeles) [inline ReturnForm.html]:
```suggestion
<input type="submit" value="$('Return book')" />
```

File: ReturnForm.html [hunk 7]
```diff
@@ -0,0 +1,9 @@
+$def with (loan)
+
+$# "do_return" is a fake "title" for the book that can be used for stats
+$ return_url = loan['book'] + '/do_return/borrow'
+
+<form class="returnbook" action="$return_url" method="post">
+<input type="submit" class="returnb" value="$('Return book')" />
```
Comment (tfmorris) [inline ReturnForm.html]:
```suggestion
<input type="submit" class="returnb" value="_$('Return book')" />
```

Missing underscore(_)

File: colors.less [hunk 8]
```diff
@@ -0,0 +1,115 @@
+// White
```
Comment (tfmorris) [inline colors.less]: Was this file included by mistake?

File: page-user.less [hunk 9]
```diff
@@ -0,0 +1,134 @@
+/**
```
Comment (tfmorris) [inline page-user.less]: Another mistaken file?

File: colors.less (hunk 8 above)
Comment (the-nine-tail) [reply colors.less]: No this file is used since I have applied css colors to button

File: page-user.less (hunk 9 above)
Comment (the-nine-tail) [reply page-user.less]: No, css is applied here line 93-130

**Comment by the-nine-tail** [comment]
On commit it gave me this error --->
To https://github.com/the-nine-tail/openlibrary.git
 ! [rejected]            HEAD -> add-loan-action-ui-fix-3186 (non-fast-forward)
error: failed to push some refs to 'https://github.com/the-nine-tail/openlibrary.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Tried to hard reset ad pull with master, still gives this error

File: static/css/less/colors.less [hunk 10]
```diff
@@ -84,6 +86,10 @@
 @white: hsl(0, 0%, 100%);
 @gray-a19b9e: hsl(330, 3%, 62%);
 @gray-d1cfd0: hsl(330, 2%, 82%);
+@grey-e5e5e5: hsl(0, 0%, 90%);
```
Comment (tfmorris) [inline static/css/less/colors.less]: Let's use consistent spelling for the colors. The establish spelling is on the line immediately above.

File: colors.less (hunk 8 above)
Comment (tfmorris) [reply colors.less]: Looks like the file has been removed from the commit and the new colors added to the existing file. Thanks.

File: static/css/page-user.less [hunk 11]
```diff
@@ -90,6 +90,44 @@ body#user {
...
+  position: relative;
+  top: .6em;
+  left: 3em;
+}
+.readbook {
+  -webkit-transition: .15s all ease-in-out;
+  -o-transition: .15s all ease-in-out;
+  transition: .15s all ease-in-out;
+  cursor: pointer;
+  background-color: @grey-e5e5e5;
```
Comment (tfmorris) [inline static/css/page-user.less]: The color spelling will need to be updated here too.

**Comment by mekarpeles** [comment]
@the-nine-tail thanks for this contribution!

It looks like this PR has merge conflicts which need to be resolved.

Also, for this specific issue, I'm not quite sure how to test it or what it should look like. Can you please provide a screen shot (before/after) and or what I should be doing to test?

**Comment by the-nine-tail** [comment]
Sure, Can you please specify what merge conflict, should I commit again..?

**Comment by jdlrobson** [comment]
#3186 has been resolved by other means.

**Pull request closed without merging**